packages/create-emotion-styled/types/common.d.ts

```diff
@@ -23,7 +23,7 @@ export type Themed<P extends object, T extends object> = P & { theme: T };
 
 export type StyledStatelessProps<P extends object, T extends object> =
   & P
-  & { theme?: T }
+  & { theme?: T, css?: BaseInterpolation }
```

Comment:
This shouldn’t be here since the css prop isn’t specific to styled components

Reply:
I can remove it, but then the tests in `create-emotion-styled` fail unless I add `import 'emotion';`. Would you rather that or that I remove the tests?

Reply:
Updated w/ the tests in place, let me know if you want me to remove them.

Reply:
Done, as a separate commit. Let me know if you want me to squash them.